common/src/verify.rs

```diff
@@ -1176,7 +1176,13 @@ mod test {
             .unwrap_err();
     }
 
-    // TODO: add test case where the transaction commit is invalid because it is extra-agenda transaction phase.
+    #[ignore]
+    #[test]
+    /// Test the case where the transaction commit is invalid because it is extra-agenda transaction phase.
+    // TODO: enable this test case when the extra-agenda transaction is implemented.
```

Comment:
I think you need to add document like `line 1309, line 1447` due to consistency.

Reply:
Fixed.